vocab/security/vocabulary.yml

```
@@ -207,9 +207,11 @@ property:
     range: xsd:dateTime
 
   - id: expires
-    label: Proof expiration time
-    defined_by: https://www.w3.org/TR/vc-data-integrity/#dfn-expires
-    domain: sec:Proof
+    label: Expiration time for a proof or verification method
+    defined_by: https://www.w3.org/TR/vc-data-integrity/#defn-proof-expires
+    domain:
+      - sec:Proof
+      - sec:VerificationMethod
```

According to @iherman in https://github.com/w3c/vc-data-model/issues/1262#issuecomment-1695821941
> the `domain` `expired` is set to both (i.e., the union) of `Proof` and `VerificationMethod`.

Note that the above does not set `rdfs:domain` to the _union_ of `Proof` and `VerificationMethod`, but rather to their _intersection_; that is, any entity with the property `expires` must be *both* a `Proof` _and_ a `VerificationMethod`.

For the apparently intended _union_, where an entity with the property `expires` could be *either* a `Proof` _or_ a `VerificationMethod`, `rdfs:domain` must be changed to `schema:domainIncludes`.
```suggestion
    domainIncludes:
      - sec:Proof
      - sec:VerificationMethod
```

---

@iherman would have to add this to the yml2vocab tool, as I don't think it has a `domainIncludes` property? Thoughts, @iherman?